LICENSE

```diff
@@ -199,4 +199,64 @@
    distributed under the License is distributed on an "AS IS" BASIS,
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
-   limitations under the License.
\ No newline at end of file
+   limitations under the License.
+   
+   
+=======================================================================
+Apache Pirk (incubating) Subcomponents:
+
+The Apache Pirk project contains subcomponents with separate copyright
+notices and license terms. Your use of the source code for the these
+subcomponents is subject to the terms and conditions of the following
+licenses.
+
```

Comment:
I'm confused - how do we create different LICENSE and NOTICE files for the different jars when they are built via the release plugin?